framework/src/main/java/org/tron/core/metrics/node/NodeMetricManager.java

```diff
@@ -36,8 +37,10 @@ private void setNodeInfo(NodeInfo nodeInfo) {
 
     nodeInfo.setIp(Args.getInstance().getNodeExternalIp());
 
-    ByteString witnessAddress = ByteString.copyFrom(Args.getLocalWitnesses()
-        .getWitnessAccountAddress(CommonParameter.getInstance().isECKeyCryptoEngine()));
+    byte[] witnessAccountAddress = Args.getLocalWitnesses()
+        .getWitnessAccountAddress(CommonParameter.getInstance().isECKeyCryptoEngine());
+    ByteString witnessAddress = !ByteUtil.isNullOrZeroArray(witnessAccountAddress) ? ByteString
```

Comment:
witnessAddress may be null, so it also should be considered in method `isActiveWitness`

Reply:
If `witnessAddress` is null,   `isActiveWitness` will always return false and won't throw the exception. If necessary, we can add the test:
```
  @Test
  public void testContains() {
    List<ByteString> list = new ArrayList<>();
    Assert.assertFalse(list.contains(null));
    list.add(null);
    Assert.assertTrue(list.contains(null));
  }
````